src/cli/simple/command.rs

```diff
@@ -67,75 +69,142 @@ pub async fn get_code(client: Client, address: String) -> anyhow::Result<()> {
     Ok(())
 }
 
+/// Deploy Aurora EVM smart contract.
+pub async fn deploy_aurora<P: AsRef<Path> + Send>(client: Client, path: P) -> anyhow::Result<()> {
+    let code = std::fs::read(path)?;
+    let result = match client.near().deploy_contract(code).await {
+        Ok(outcome) => match outcome.status {
+            FinalExecutionStatus::SuccessValue(_) => {
+                "Aurora EVM has been deployed successfully".to_string()
+            }
+            FinalExecutionStatus::Failure(e) => format!("Error while deployed Aurora EVM: {e}"),
+            _ => "Error: Bad transaction status".to_string(),
+        },
+        Err(e) => format!("{e:?}"),
+    };
+    println!("{result}");
+
+    Ok(())
+}
+
 /// Initialize Aurora EVM smart contract.
 pub async fn init(
     client: Client,
     chain_id: u64,
     owner_id: Option<String>,
     bridge_prover: Option<String>,
     upgrade_delay_blocks: Option<u64>,
+    custodian_address: Option<String>,
+    ft_metadata_path: Option<String>,
 ) -> anyhow::Result<()> {
+    let to_account_id = |id: Option<String>| {
+        id.map_or_else(
+            || {
+                client
+                    .near()
+                    .engine_account_id
+                    .to_string()
+                    .parse()
+                    .map_err(|e| anyhow::anyhow!("{e}"))
+            },
+            |id| id.parse().map_err(|e| anyhow::anyhow!("{e}")),
+        )
+    };
+
+    let owner_id = to_account_id(owner_id)?;
+    let prover_id = to_account_id(bridge_prover)?;
+
+    // Init Aurora EVM.
     let args = NewCallArgs {
         chain_id: H256::from_low_u64_be(chain_id).into(),
-        owner_id: owner_id
-            .and_then(|id| AccountId::try_from(id).ok())
-            .unwrap_or_default(),
-        bridge_prover_id: bridge_prover
-            .and_then(|id| AccountId::try_from(id).ok())
-            .unwrap_or_default(),
+        owner_id,
+        bridge_prover_id: prover_id.clone(),
         upgrade_delay_blocks: upgrade_delay_blocks.unwrap_or_default(),
     }
     .try_to_vec()?;
 
-    let result = client.near().contract_call("new", args).await;
+    match client.near().contract_call("new", args).await?.status {
+        FinalExecutionStatus::Failure(e) => {
+            anyhow::bail!("Error while initialized Aurora EVM: {e}")
+        }
+        FinalExecutionStatus::Started | FinalExecutionStatus::NotStarted => {
+            anyhow::bail!("Error while initialized Aurora EVM: Bad status of the transaction")
+        }
+        FinalExecutionStatus::SuccessValue(_) => {}
+    }
 
-    println!("{result:?}");
+    // Wait for block committing.
+    tokio::time::sleep(tokio::time::Duration::from_secs(1)).await;
```

Comment:
This is just to avoid the issue with getting the current nonce right? I think there are two options to solve this other than using a sleep (using sleep works, but it makes the CLI feel more sluggish than it needs to be):

1. manually control the nonce (this is how I do it in the advanced interface, though it being in the other interface is not relevant because the nonce is still not surfaced to the user)
2. make a single batch transaction with both `new` and `new_eth_connector` calls in it so that you only need to send one transaction instead of two